openapi.yaml

```
@@ -10185,20 +10185,126 @@ paths:
           linode-cli networking ip-update \
             97.107.143.141 \
             --rdns "test.example.org"
-  /networking/ipv4/assign:
+  /networking/ips/assign:
     x-linode-cli-command: networking
     post:
       x-linode-grant: read_write
       tags:
       - Networking
       summary: Linodes Assign IPs
+      description: |
+        Assign multiple IPv4 addresses and/or IPv6 ranges to multiple Linodes in one Region. This allows
+        swapping, shuffling, or otherwise reorganizing IPs to your Linodes.
+
+        The following restrictions apply:
+        * All Linodes involved must have at least one public IPv4 address after assignment.
+        * Linodes may have no more than one assigned private IPv4 address.
+        * Linodes may have no more than one assigned IPv6 range.
+        * [Open a Support Ticket](/docs/api/support/#support-ticket-open) to request expansion of these restrictions.
```

I don't believe the first restriction can be altered, would it be better to be more specific here?
```suggestion
        * [Open a Support Ticket](/docs/api/support/#support-ticket-open) to request additional IPv4 addresses or IPv6 ranges.
```

---

yes, that's definitely clearer -- nice catch